packages/amplify-migration-tests/src/__tests__/migration_tests_v10/__snapshots__/api-rest-lambda.migration.test.ts.snap

```diff
@@ -3,6 +3,56 @@
 exports[`api lambda migration tests ...adds lambda with multiple rest APIs with v10 and pulls without drift in latest version 1`] = `
 "Resources
 [~] AWS::Lambda::Function UserPoolClientLambda 
+ ├─ [~] Code
+ │   └─ [~] .ZipFile:
+ │       ├─ [-] const response = require('cfn-response');
+const aws = require('aws-sdk');
+const identity = new aws.CognitoIdentityServiceProvider();
+exports.handler = (event, context, callback) => {
+  if (event.RequestType == 'Delete') {
+    response.send(event, context, response.SUCCESS, {});
+  }
+  if (event.RequestType == 'Update' || event.RequestType == 'Create') {
+    const params = {
+      ClientId: event.ResourceProperties.clientId,
+      UserPoolId: event.ResourceProperties.userpoolId,
+    };
+    identity
+      .describeUserPoolClient(params)
+      .promise()
+      .then(res => {
+        response.send(event, context, response.SUCCESS, { appSecret: res.UserPoolClient.ClientSecret });
+      })
+      .catch(err => {
+        response.send(event, context, response.FAILED, { err });
+      });
+  }
+};
+
+ │       └─ [+] const response = require('cfn-response');
+const aws = require('aws-sdk');
+const identity = new aws.CognitoIdentityServiceProvider();
+exports.handler = (event, context, callback) => {
+  if (event.RequestType == 'Delete') {
+    response.send(event, context, response.SUCCESS, {});
+  }
+  if (event.RequestType == 'Update' || event.RequestType == 'Create') {
+    const params = {
+      ClientId: event.ResourceProperties.clientId,
+      UserPoolId: event.ResourceProperties.userpoolId,
+    };
+    identity
+      .describeUserPoolClient(params)
+      .promise()
+      .then((res) => {
+        response.send(event, context, response.SUCCESS, { appSecret: res.UserPoolClient.ClientSecret });
+      })
+      .catch((err) => {
+        response.send(event, context, response.FAILED, { err });
+      });
+  }
+};
+
```

Comment:
Drifts in this file are expected. Therefore recording them.
They come from re-formatted lambdas in auth category.
See https://github.com/aws-amplify/amplify-cli/pull/12028#discussion_r1110030827 for reference.